modules/text_server_adv/text_server_adv.h

```diff
@@ -158,6 +158,7 @@ class TextServerAdvanced : public TextServerExtension {
 	// ICU support data.
 
 	static bool icu_data_loaded;
+	static PackedByteArray icu_data;
```

Comment:
Does it need to be cleared manually in the destructor, or this is implicit?

Reply:
It's `PackedByteArray`, so it will be freed on exit.